UVa/BasicRemains.java

```diff
@@ -0,0 +1,41 @@
+
+/*
+Given a base b and two non-negative base b integers
```

Comment:
Could you please fix these comments to follow either javadoc style or multiline comment stye:

/**
 * commet here
 * comment here
 */

Note that the only difference between a javadoc and a multiline comment is that a javadoc starts with /** while a comment starts with /*